### ISSUE TITLE
Follow up to API effort

### ISSUE THREAD

**Issue by rraustad** [NREL/EnergyPlus#7803]
Issue overview
--------------
Followup to #7776. See PR discussion for more information. 
 
 - [x] Find all TODOs in src/EnergyPlus/api
 - [x] Fix actuators and sensors for all real/bool/int types
 - [x] Documentation
 - [x] API documentation -- not sure of location yet
 - [x] Gracefully handle running a python case in a dev build without python linked
 - [x] Set PYTHONHOME in the plugin manager: [doc](https://docs.python.org/3/c-api/init.html#c.Py_SetPythonHome)
 - [x] Include API header files in <eplusinstall>/include
 - [x] Add note in readme about API being a preliminary thing and subject to small breaking changes initially
 - [x] Add "linked-with-python-version" output to EnergyPlus `--help` or maybe `--version` message
 - [x] Use Doxygen to build API docs?
 - [x] Reorder source files in `src/EnergyPlus/CMakeLists.txt`
 - [x] Check other actuator types (logical, int)
 - [x] Fatal out if plugin is found in input during an API based run
 - [x] Add Python license to install
 - [x] Change EnergyPlusAPI.hh to just .h

**Comment by Myoldmopar** [comment]
On the documentation, there will be a couple things:
 - Documentation of the Python Plugin (EMS) System embedded heavily in the EMSApplicationGuide.
 - The API Application Guide could be a separate document, not sure.

**Comment by Myoldmopar** [comment]
Running the CI-built Mac package leads to this in the E+ output:

```
EnergyPlus Starting
EnergyPlus, Version 9.3.0-cc126b865c, YMD=2020.02.23 09:50
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Initializing Response Factors
Calculating CTFs for "INTERIORFURNISHINGS", Construction # 1
```

But setting PYTHONHOME cleans this up.  The plugin manager needs to set PYTHONHOME when setting PYTHONPATH as well, it seems, on Mac.

**Comment by Myoldmopar** [comment]
But huge news though, I had CI build package from the latest develop.  I then pulled all three platforms onto totally separate machines and ran them, completely successfully.  The plugin system worked.  The only issue at all is the little message to clean up I mentioned a few minutes ago.

**Comment by Myoldmopar** [comment]
Many things are being addressed in #7836.  However, some things are almost certainly going to be pushed to a future release.  These should be tracked in #7877.
